src/test/java/it/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSourceIT.java

```diff
@@ -147,6 +154,9 @@ public void testCreateSCM() {
         String credentialsId = bbJenkinsRule.getBbAdminUsernamePasswordCredentialsId();
         String id = randomUUID().toString();
         String serverId = serverConf.getId();
+        Set<SCMHeadCategory> categorySet = Arrays.stream(new SCMHeadCategory[]{UncategorizedSCMHeadCategory.DEFAULT,
```

Comment:
May I propose `Set.of(new SCMHeadCategory[]{UncategorizedSCMHeadCategory.DEFAULT,new ChangeRequestSCMHeadCategory(Messages._bitbucket_scm_pullrequest_display()))` instead?

We're in Java 11 now, so we can use these new methods now :)